tests/bemioTest.m

```diff
@@ -0,0 +1,109 @@
+classdef bemioTest < matlab.unittest.TestCase
+    
+    properties
+        wsDir = '';
+        testDir = '';
+        bemioDir = '';
+        wamitDir = '';
+        nemohDir = '';
+        capytaineDir = '';
+        aqwaDir = '';
+    end
+    
+    methods (Access = 'public')
+        
+        function obj = bemioTest
+            
+            % Set WEC-Sim, test and BEMIO example directories
+            obj.testDir = fileparts(mfilename('fullpath'));
+            obj.wsDir = fullfile(obj.testDir,'..');
+            obj.bemioDir = fullfile(obj.wsDir,'examples\BEMIO\');
+            obj.wamitDir = fullfile(obj.bemioDir,'WAMIT\');
+            obj.nemohDir = fullfile(obj.bemioDir,'NEMOH\');
+            obj.capytaineDir = fullfile(obj.bemioDir,'CAPYTAINE\');
+            obj.aqwaDir = fullfile(obj.bemioDir,'AQWA\');
+            
+            % Hide figures
+            set(0,'DefaultFigureVisible','off')
+            
+        end
+        
+    end
+    
+    methods(TestMethodTeardown)
+        
+        function closePlotsHydro(testCase)
+            close(waitbar(0));
+            close all
+            clear hydro
+            set(0,'DefaultFigureVisible','off')
+        end
+        
+    end
+    
+    methods(TestClassTeardown)
+        
+        function cdToTestDir(testCase)
+            cd(testCase.testDir);
+        end
+        
+    end
+    
+    methods(Test)
+        
+        function combine_bem(testCase)
+
+            cd(fullfile(testCase.capytaineDir,'Cylinder'))
+            
+            hydro = struct();
+            hydro = Read_CAPYTAINE(hydro,'.\cylinder_full.nc');
+            hydro = Read_NEMOH(hydro,'..\..\NEMOH\Cylinder\');
+            hydro(end).body = {'cylinder_nemoh'};
+            hydro = Read_WAMIT(hydro,'..\..\WAMIT\Cylinder\cyl.out',[]);
+            hydro(end).body = {'cylinder_wamit'};
+            hydro = Combine_BEM(hydro);
```

Comment:
It is better to skip this test as a known error, rather than leave it as a failure. The automated CI is always going to fail if we leave it like this. See https://github.com/akeeste/WEC-Sim/pull/8 for the approach to skip this test based on a known failure

Reply:
I agree the test should be skipped if the fix is not already present. However since this PR is only merging into dev, the fix will be present and the test will pass as soon as the PR is merged. To simplify this I pulled dev into this branch so that all tests are passing before the merge.

Reply:
Oh, I see. Shouldn't we be putting this into master as well?

Reply:
I assumed this wouldn't happen until the entire dev branch is merged into master and a new release is tagged. I believe v4.3 was planned for the near future.